spec/models/ext_management_system_spec.rb

```diff
@@ -396,6 +396,91 @@
     end
   end
 
+  context "#pause!" do
+    it 'disables an ems with child managers and moves parent to maintenance zone' do
+      zone  = FactoryGirl.create(:zone)
+      ems   = FactoryGirl.create(:ext_management_system, :zone => zone)
+      child = FactoryGirl.create(:ext_management_system, :zone => zone)
+      ems.child_managers << child
+
+      ems.pause!
+
+      expect(ems.enabled).to be_falsy
+      expect(ems.zone).to eq(Zone.maintenance_zone)
+      expect(ems.zone_before_pause).to eq(zone)
+
+      child.reload
+      expect(child.enabled).to be_falsy
+    end
+  end
+
+  context "#resume" do
+    it "enables an ems with child managers and move parent from maintenance zone" do
+      zone = FactoryGirl.create(:zone)
+      ems = FactoryGirl.create(:ext_management_system,
+                               :zone_before_pause => zone,
+                               :zone              => Zone.maintenance_zone,
+                               :enabled           => false)
+
+      child = FactoryGirl.create(:ext_management_system,
+                                 :zone    => zone,
```

Comment:
This should also be the maintenance zone

Reply:
:+1: agreed and should confirm below that the child manager is moved out of the maintenance zone after the parent's `resume!`

Reply:
I would still feel better if we called `#pause!` and `#resume!` instead of `#disable!` and `#enable!` and rely on the callbacks.  It isn't that hard for someone to forget and not use that mixin then this will be broken.

I put in a commit https://github.com/agrare/manageiq/commit/dce89761595ed14b431afacf0740060d63ce640d with what I think would be sufficient tests, you can cherry-pick if you want or just take what you need.